model.py

```diff
@@ -5209,6 +5209,15 @@ def for_foreign_id(self, _db, data_source, foreign_id_type, foreign_id, rights_s
         if was_new and not license_pool.availability_time:
             now = datetime.datetime.utcnow()
             license_pool.availability_time = now
+
+        if was_new:
+            # Set the LicensePool's initial values to indicate
+            # that we don't actually know how many copies we own.
+            license_pool.licenses_owned = 0
+            license_pool.licenses_available = 0
+            license_pool.licenses_reserved = 0
+            license_pool.patrons_in_hold_queue = 0
+
```

Comment:
Following a discussion with Amy and Darya we decided that it's important to have LicensePools even when we don't know how many licenses we own, because LicensePools also contain format information. Rather than risk the existence of LicensePools whose circulation information is None, we start with the conservative estimate that we know about the book but own no copies.